tests/models/test_user.py

```diff
@@ -124,6 +124,27 @@ def test_contributed_property(self):
         projects_contributed_to = project.model.Node.find(Q('contributors', 'eq', self.user._id))
         assert_equal(list(self.user.contributed), list(projects_contributed_to))
 
+    def test_contributor_to_property(self):
+        # Make sure there's at least one deleted project and bookmark collection
+        factories.ProjectFactory(creator=self.user, is_deleted=True)
+        factories.BookmarkCollectionFactory(creator=self.user)
+
+        contributor_to = project.model.Node.find(
```

Comment:
This is not an ideal approach. Rather than re-creating the query, set up a node with a mix of visible and non-visible contributors and ensure that the correct contributors are in the correct lists, and the ones who aren't supposed to be are not.